src/main/webapp/WEB-INF/tags/instructor/search/searchPageInput.tag

```diff
@@ -1,6 +1,6 @@
 <%@ tag description="instructorSearch.jsp - Instructor search page input" pageEncoding="UTF-8" %>
-<%@ tag import="teammates.common.util.Const"%>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
Revert these unnecessary changes

Reply:
Done, however there is some inconsistency in it throught this and other JSP files - some tags have spaces at the end, some do not.